api/data_refinery_api/tests.py

```diff
@@ -342,320 +328,42 @@ def test_compendia(self):
         drc1.s3_key = "drc2.tsv"
         drc1.save()
 
-        response = self.client.get(reverse('compendia'))
-        response_json = response.json()
+        response = self.client.get(reverse('computed-files'), {'is_compendia': True})
+        response_json = response.json()['results']
         self.assertEqual(3, len(response_json))
         # Prove that the download_url field is missing and not None.
-        self.assertEqual('NotPresent', response_json[0].get('download_url', 'NotPresent'))
+        self.assertEqual(None, response_json[0].get('download_url', None))
 
         # We don't actually want AWS to generate a temporary URL for
         # us, and it won't unless we're running in the cloud, but if
         # we provide an API Token and use the WithUrl serializer then
         # it will set the download_url field to None rather than
         # generate one.
 
-        # Get a token first
-        response = self.client.get(reverse('token'),
-                                    content_type="application/json")
+        # Create a token first
+        response = self.client.post(reverse('token'), content_type="application/json")
         token = response.json()
         token['is_activated'] = True
         token_id = token['id']
-        response = self.client.post(reverse('token'),
+        response = self.client.put(reverse('token_id', kwargs={'id': token_id}),
                                     json.dumps(token),
                                     content_type="application/json")
 
-        response = self.client.get(reverse('compendia'), HTTP_API_KEY=token_id)
-        response_json = response.json()
+        response = self.client.get(reverse('computed-files'), {'is_compendia': True}, HTTP_API_KEY=token_id)
+        response_json = response.json()['results']
         self.assertEqual(3, len(response_json))
         self.assertIsNone(response_json[0]['download_url'])
 
-    def test_search_and_filter(self):
```

Comment:
Why was this test removed?

Reply:
We changed the search and started using elastic search. There are tests for that here https://github.com/AlexsLemonade/refinebio/blob/c67e006cc524ad9f57119d5a7f96f0f9a333f6bd/api/data_refinery_api/tests.py#L864

Also I wasn't able to make this one work with elastic search, seems to be tricky.